### ISSUE TITLE
Fix SLF4J warnings in console when running integration tests in IntelliJ

### ISSUE THREAD

**Issue by lptr** [gradle/gradle#6462]
When running an integration test (e.g. `BuildScanConfigIntegrationTest.enabled and disabled are false with no flags`) currently the following is displayed in IntelliJ:

```text
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [file:/Users/lptr/Workspace/gradle/gradle/out/production/logging/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/Users/lptr/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-simple/1.7.16/f0cacc3d21e1027c82006a7a6f6cf2cad86d2a4f/slf4j-simple-1.7.16.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.gradle.internal.logging.slf4j.OutputEventListenerBackedLoggerContext]
> Task :compileJava NO-SOURCE

> Task :compileGroovy
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [file:/Users/lptr/Workspace/gradle/gradle/out/production/logging/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/Users/lptr/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-simple/1.7.16/f0cacc3d21e1027c82006a7a6f6cf2cad86d2a4f/slf4j-simple-1.7.16.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.gradle.internal.logging.slf4j.OutputEventListenerBackedLoggerContext]
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.vmplugin.v7.Java7$1 (file:/Users/lptr/.gradle/caches/modules-2/files-2.1/org.codehaus.groovy/groovy/2.5.2/ff667436cfd98cc2c7e58631883c75d23d52261b/groovy-2.5.2.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class,int)
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.vmplugin.v7.Java7$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

All that noise about SLF4J bindings makes it harder to comprehend the output of the test.

**Comment by lptr** [comment]
Fixing the Java warnings would be another nice touch.

**Comment by ldaley** [comment]
Related to #1893

**Comment by lptr** [comment]
I understand why we have the second binding, but why do we have SLF4J's own classes in our IntelliJ `out` directory?

```text
SLF4J: Found binding in [file:/Users/lptr/Workspace/gradle/gradle/out/production/logging/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/Users/lptr/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-simple/1.7.16/f0cacc3d21e1027c82006a7a6f6cf2cad86d2a4f/slf4j-simple-1.7.16.jar!/org/slf4j/impl/StaticLoggerBinder.class]
```

**Comment by ldaley** [comment]
You’ll find a version of that class in our logging module. This is how slf4j bootstraps.

**Comment by lptr** [comment]
Right. So it's the one coming in via `slf4j-simple` that is superfluous. It gets into the mix via `org.gradle:sample-check:0.4.2`: https://e.grdev.net/s/zmdseyz7jsvgi/dependencies?dependencies=slf4j-simple&expandAll&focusedDependency=WzQwLDEsODg2LFszLDAsWzZdXV0&focusedDependencyView=dependents&toggled=W1swLDBdLFswLDBdXQ

**Comment by lptr** [comment]
Should be fixed by Exemplar 0.5.0.

**Comment by wolfs** [comment]
Fixed by the exemplar update.
